src/services/session.js

```diff
@@ -27,7 +27,38 @@ const getAll = () => http.get('sessions').then(sessions => sessions.reverse());
  */
 const getCurrent = () => http.get('sessions/current');
 
+/**
+ * Get days left in the term
+ * @return {Promise.<Session>}
```

Comment:
This function does not return a Promise resolved by a `Session`. Can we create a new `@typedef` like the one above, and have it define what the object returned by this function will look like?